ports/soh2/Ship of Harkinian 2.sh

```diff
@@ -90,11 +123,11 @@ fi
 
 # Run the game
 echo "Loading, please wait... (might take a while!)" > $CUR_TTY
-$GPTOKEYB "2s2h.elf" -c "soh2.gptk" &
-pm_platform_helper 2s2h.elf
+$GPTOKEYB "2s2h.elf" -c "soh2.gptk" & 
```

Comment:
Maybe there is a good reason to remove it, just tell me :)
```suggestion
pm_platform_helper "2s2h.elf"
$GPTOKEYB "2s2h.elf" -c "soh2.gptk" & 
```